src/components/graph-view.js

```diff
@@ -674,84 +729,126 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
       };
       this.setState(newState);
 
-      if (onSelectEdge) {
-        onSelectEdge(this.state.edges[originalArrIndex]);
-      }
+      onSelectEdge && onSelectEdge(this.state.edges[originalArrIndex]);
+      onSelect && onSelect({ nodes: [], edges: [] });
```

Comment:
Just thinking - what's the reason for not passing edge into `edges: []` (or node into `nodes: []`)? This would allow our app to only consume onSelect when we switch to multi-select.

Reply:
Thanks. I added the selected edge to the edges array.